cockatrice/src/tab_server.cpp

```diff
@@ -71,7 +71,20 @@ void RoomSelector::processListRoomsEvent(const Event_ListRooms &event)
     const int roomListSize = event.room_list_size();
     for (int i = 0; i < roomListSize; ++i) {
         const ServerInfo_Room &room = event.room_list(i);
-        
+
+        /*
+        * A room can have a permission level and a privilege level. How ever we want to display only the necessary information 
+        * on the server tab needed to inform users of required permissions to enter a room. If the room has a privilege level
+        * the server tab will display the privilege level in the "permissions" column in the row however if the room contains
+        * a permissions level for the room the permissions level defined for the room will be displayed.
+        */
+
+        QString roomPermissionDisplay = QString::fromStdString(room.privilegelevel()).toLower();
```

Comment:
It will set the roomPermissionDisplay to what the privlevel definition of the room is (none,privileged,vip,donator at this time).  The next line then trumps it if there is a permission value defined.